### ISSUE TITLE
pixi-flump-runtime broken ? 

### ISSUE THREAD

**Issue by yordan-kanchelov** [jackwlee01/pixi-flump-runtime#25]
From the dog example which I modified a little to see the basic methods and properties on the movie ,the only one that is working as expected is animationSpeed. 

The movie is movie is starting from zero frame moving and looping. Any help would be appreciated.
```
var stage = new PIXI.Stage(0x66FF99);
var renderer = new PIXI.autoDetectRenderer(480, 270);
var movie;
document.body.appendChild(renderer.view);


var loader = new PIXI.loaders.Loader();
loader.after(PIXI.flump.Parser(1));
loader.add("DogLibrary", "./flump-assets/dog/library.json");
loader.once("complete", begin.bind(this));
loader.load();

animate();


function begin(){
	movie = new PIXI.flump.Movie("TestScene");	
	
	movie.loop = false;	
	movie.animationSpeed = 0.5;
	movie.gotoAndPlay(15);
        movie.stop();
	stage.addChild(movie);
}


function animate(){
	 requestAnimationFrame(animate);
	 renderer.render(stage);
}
```

**Comment by NicolasVereenoghe** [comment]
Hello,
I tested this code, and it seems to work as expected, from the json and atlas dog files provided as example.

I cannot open the fla source because I only have flash cs5, but it seems the TestScene MovieClip is stopped (the red circle doesn't scroll anymore).

It works like in flash :
if you stop a movie clip, its timeline does stop, but all contained movie clip continue to play, this is what I experienced here : red spot stops, I suppose it is animated on the TestScene timeline, but the contained movie clips still continue to play.

The "movie.loop = false" statement doesn't make sense because you stop the movie timeline.

You would need some recurive méthods to apply "stop" / "loop = false" on every contained movie clip. Like this :
https://github.com/NicolasVereenoghe/git.vega.pixi4.flump/blob/master/src/vega/utils/UtilsFlump.hx#L213-L279
